data/lescure2020clinical/lescure2020clinical-extraction.md

```diff
@@ -0,0 +1,85 @@
+# Extraction of the raw data for Lescure et al. (2020) using the data published by Goyal et al. (2020)
+
+[Goyal et al. (2020)](https://www.science.org/doi/10.1126/sciadv.abc7112) developed mathematical models to project multiple therapeutic approaches. The authors used four datasets of SARS-CoV-2 shedding in the absence of effective treatment to develop and validate a mathematical model. These data included including a dataset of 4 patients from France ([Lescure et al. 2020](https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(20)30200-0/fulltext)).
+
+First, we `import` python modules needed:
+
+```python
+#import modules;
+import yaml
+import pandas as pd
+
+#functions to add folded blocks and literal blocks;
+class folded_str(str): pass
+class literal_str(str): pass
+
+def folded_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='>')
+def literal_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='|')
+
+yaml.add_representer(folded_str, folded_str_representer)
+yaml.add_representer(literal_str, literal_str_representer)
+```
+
+Raw data ([Viral_Loads](https://github.com/shedding-hub/shedding-hub/blob/main/data/lescure2020clinical/Viral_Loads.csv)), which is stored on [Shedding Hub](https://github.com/shedding-hub), will be loaded and cleaned to match the most updated [schema](https://github.com/shedding-hub/shedding-hub/blob/main/data/.schema.yaml).
+
+```python
+#load the data;
+Lescure2020 = pd.read_csv("Viral_Loads.csv")
+#Subset the data for Lescure et al. 2020;
+Lescure2020 = Lescure2020[Lescure2020["cov_study"]==4]
+#correct some doa errors;
+Lescure2020.loc[(Lescure2020["ID"]=="E3") & (Lescure2020["dao"]==13),"dao"]=14
```

Comment:
What are these corrections?

Reply:
The last two records had incorrect day (13 adn 15) based on Figure 2 in Lescure et al. (2020). We corrected them with days 14 and 16. I also added this explanation in the comment.